### ISSUE TITLE
a11y: Increase Color Contrast for text and icons

### ISSUE THREAD

**Issue by ambersdai** [openstax/cnx#116]
## Overview
From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

### Issue 1: Color Issues
The color contrast is not sufficient (1.4.3).
Examples (insufficient color contrast): A regular-sized text and sometimes a large-sized text does not have a good
contrast with its background

### Reference to standards
1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1,
except for the following (Level AA):
- Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
- Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
- Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.
### Suggestion
Change the color of the text/background so that it has sufficient contrast.

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df65413267482d4771ca3/f414219322642d825ffff681c9f7aeb0/image.png)

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df65413267482d4771ca3/0539978ea71ef824c0886fe082038335/image.png)

## PRs
What PRs need to be merged to complete this task/feature
## Test Plan
How will QA know this feature works? What parts of regression tests would be most affected?
## Acceptance Criteria
Expected behavior
